sonar/modules/documents/utils.py

```diff
@@ -81,6 +83,8 @@ def get_file_links(file, record):
     :param record: Record.
     :returns: Dict containing the URL, the download URL and the type of link.
     """
+    key = quote(unicodedata.normalize('NFD', file['key']))
```

Comment:
I would not do the normalization here but on the final url. A better approach is to use a decorator to do it for several functions or methods i.e. get_thumbnail